Mathlib/Combinatorics/SimpleGraph/Hamiltonian.lean

```diff
@@ -72,10 +72,7 @@ lemma IsHamiltonianCycle.isCycle (hp : p.IsHamiltonianCycle) : p.IsCycle :=
 
 lemma IsHamiltonianCycle.map {H : SimpleGraph β} (f : G →g H) (hf : Bijective f)
     (hp : p.IsHamiltonianCycle) : (p.map f).IsHamiltonianCycle where
-  toIsCycle := by
-    rw [map_isCycle_iff_of_injective]
-    exact isCycle hp
-    exact Bijective.injective hf
+  toIsCycle := isCycle.map hf.injective hp.isCycle
```

Comment:
```suggestion
  toIsCycle := hp.isCycle.map hf.injective 
```
once you fix the name

Reply:
Ah, I see what is going on now, thanks :).